creme/utils/shift.py

```diff
@@ -0,0 +1,121 @@
+import collections
+
+
+class Shift:
+    """Shift the observations with the selected period.
+
+    Shift class allows to store and recall at regular intervals values observed in a stream.
+    The method ```get()`` returns the ```n-th - period``` observed entry.
+
+    Parameters:
+        period (int): Number of periods to shift. Must be positive.
+        missing: Handle missing values when there is'nt any reference for a given period.
```

Comment:
You should instead say: "value to output when less than `period` values have been observed".